quickwit/quickwit-control-plane/src/model/mod.rs

```diff
@@ -196,7 +196,7 @@ impl ControlPlaneModel {
     ) -> ControlPlaneResult<()> {
         let index_metadata = self.index_table.get_mut(index_uid).ok_or_else(|| {
             MetastoreError::NotFound(EntityKind::Index {
-                index_id: index_uid.to_string(),
+                index_id: index_uid.index_id().to_string(),
```

Comment:
I didn't want to waste time creating an issue for this bug I found so I fixed it here myself. Please lemme know if your intentions where to keep `index_uid` then I will revert it.

Reply:
That's fine.